### ISSUE TITLE
[Question] RetryWithErrorOutputParser with_structured_output

### ISSUE THREAD

**Issue by austinmw** [langchain-ai/langchain-aws#155]
Hi, any idea if langchain's `RetryWithErrorOutputParser` is compatible with `llm.with_structured_output()`? Since LLMs can return incorrect responses that would cause Pydantic parsing to fail, it would be good to provide the previous output and error to the LLM when doing retries.

**Comment by adveatkarnik1** [comment]
Hi, did you get any clarity on this?

**Comment by austinmw** [comment]
Nope. Handled failures with LangGraph, but would be nice to have this built in.
